mantisshrimp/datasets/pets/transforms.py

```diff
@@ -0,0 +1,59 @@
+__all__ = [
+    "aug_tfms_albumentations_pets"
+]
+
+from mantisshrimp.imports import *
+from mantisshrimp.core import *
+from mantisshrimp.transforms.albu_transform import *
+from mantisshrimp.utils.utils import *
+
+import albumentations as A
```

Comment:
Can you move this to inside the function? Or else it will be a requirement to have albumentations to use pets